### PR TITLE
Edit ingest management troubleshooting docs

### PR DESCRIPTION
**Pull request by dedemorton**
Please answer all questions flagged for reviewers.

Summary of changes:

* Edit to conform to Elastic style guidelines
* Break walls of text into easy-to-scan list
* Use attributes for product names

Ooof. The diff is hard to follow. It's probably best to review the preview: https://stack-docs_1160.docs-preview.app.elstc.co/guide/en/ingest-management/master/ingest-management-troubleshooting.html

### CODE REVIEW

**Comment by dedemorton** [comment]
cc @bmorelli25 Here's my quick and dirty edit in case you have a few spare cycles and want to add your thoughts. Motto: progress over perfection.

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 1]
```diff
@@ -78,81 +192,125 @@ elastic-agent.exe run
...
+If {elastic-agent} is set up and running, but you don't see data in {kib}:
 
-[float]
-[[i-deleted-my-agent]]
-== I deleted {agent} in the {fleet} {ingest-manager} app how can I get it back?
+//REVIEWERS: Is there a better command that we should recommend to users here?
+//I think it's good to show an actual command because we could have a newbie
+//reading this topic.
 
-Its ok, we got your back!  The data is still in {es}. And get the {agent} back in the UI, you will need only to stop, then re-enroll the {agent} on the host and run it again.
```
Comment (EricDavisX) [inline docs/en/ingest-management/troubleshooting.asciidoc:105]: I was trying so hard to be Elastic and put some humor in, I'm ok if you want to leave it out tho. Minimal sad panda tho.  About the command, its the same command ./elastic-agent run that is listed above we can re-cite it (or put it to a variable if its going to be repeated more) but that detail is up to you.

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 2]
```diff
@@ -78,81 +192,125 @@ elastic-agent.exe run
...
+. Make sure there are no errors in {kib}. If there are errors, resolve them.
```
Comment (EricDavisX) [inline docs/en/ingest-management/troubleshooting.asciidoc:233]: actually a known issue in the product is that kibana is showing an error when an agent is shut down, tho it is not really an 'error' state.  we may not want to explicitly say this as it can confuse users?  The start up option they need to do to get it back is on the host tho.  as noted above.

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 3]
```diff
@@ -4,72 +4,186 @@
...
 [source,yaml]
 ----
 xpack.ingestManager.enabled: true
+xpack.ingestManager.fleet.tlsCheckDisabled: true <1>
+xpack.security.enabled: true
+elasticsearch.username: "elastic" <2>
+elasticsearch.password: "abc123iUnbRftkABC123"
 ----
+<1> This setting is not required if you configure TLS checking.
+<2> Specify a user who is authorized to use {ingest-manager}
```
Comment (EricDavisX) [inline docs/en/ingest-management/troubleshooting.asciidoc:108]: specifically the user needs to have 'superuser' privilege.  which is a little hectic, and only for the Alpha release (we hope).

Comment (dedemorton) [reply docs/en/ingest-management/troubleshooting.asciidoc:108]: Yah, I was conflicted about whether to cover it here because we mention that elsewhere.

Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:108]:
```suggestion
<2> Specify a user who is authorized to use {ingest-manager}.
```

File: docs/en/ingest-management/troubleshooting.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/en/ingest-management/troubleshooting.asciidoc:105]: Oh I know. I kept the comment...it's just a bit lower down. We just want to be a little careful about humor in troubleshooting topics. I think it works, tho, so let's go with it.

Comment (dedemorton) [reply docs/en/ingest-management/troubleshooting.asciidoc:105]: About the comment...I was referring to the API call, not a command. My bad. I'll leave it as-is since you didn't flag it. Thanks!

File: docs/en/ingest-management/troubleshooting.asciidoc (hunk 2 above)
Comment (dedemorton) [reply docs/en/ingest-management/troubleshooting.asciidoc:233]: aha...I will delete the line. In that case, I don't think we should mention errors in the intro to this section either.

Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:233]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 4]
```diff
@@ -78,81 +192,124 @@ elastic-agent.exe run
...
 
-I think I have {agent} set up and running and there are no errors on the agent logs and I still don't see data in {kib}. The recommended way to assess this is to go straight to {es} data queries, or maybe just use {kib} Discover app. Look for any data in the index your {agent} is sending to (metrics-* for example) and see if the name.host of your host machine shows. If not, then perhaps the data is getting trapped / blocked in the network or isn't sent off of the {agent} host correctly in the first place due to some Firewall or security reason. While it should be redundant, it is all the same potentially helpful to install the stand-alone Metricbeat (see the Elastic docs for information) to see if that works to send data into ES, which would indicate a bug or problem in the {agent} side.
+If {elastic-agent} is set up and running, but you don't see data in {kib}:
 
-[float]
-[[i-deleted-my-agent]]
-== I deleted {agent} in the {fleet} {ingest-manager} app how can I get it back?
+//REVIEWERS: Is there a better command that we should recommend to users here?
+//I think it's good to show an actual command because we could have a newbie
+//reading this topic.
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:231]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 5]
```diff
@@ -4,72 +4,186 @@
...
+* https://github.com/elastic/beats/issuess[{beats}]
+* https://github.com/elastic/package-registry/issues[{package-registry}]
+
+Contact us https://ela.st/ingest-manager-feedback[here]. Your feedback is very
+valuable to us.
+
+//REVIEWERS: Using "here" for link text is strongly discouraged because it's
+//bad for SEO. What text should we provide instead? 
+
+// REVIEWERS: Do you think it's worthwhile to separate the jump list into two
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:25]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 6]
```diff
@@ -4,72 +4,186 @@
...
+Contact us https://ela.st/ingest-manager-feedback[here]. Your feedback is very
+valuable to us.
+
+//REVIEWERS: Using "here" for link text is strongly discouraged because it's
+//bad for SEO. What text should we provide instead? 
+
+// REVIEWERS: Do you think it's worthwhile to separate the jump list into two 
+// sections? We could create separate topics with a jump list at the top, but
+// then users can't search the page. In general, presenting a list of more than
+// 8 options makes the info harder for users to process.
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:28]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 7]
```diff
@@ -4,72 +4,186 @@
...
+valuable to us.
+
+//REVIEWERS: Using "here" for link text is strongly discouraged because it's
+//bad for SEO. What text should we provide instead? 
+
+// REVIEWERS: Do you think it's worthwhile to separate the jump list into two 
+// sections? We could create separate topics with a jump list at the top, but
+// then users can't search the page. In general, presenting a list of more than
+// 8 options makes the info harder for users to process. 
+
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:29]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 8]
```diff
@@ -4,72 +4,181 @@
...
-We have collected the most common known problems below. If your question isn't answered here, please review open issues in the https://github.com/elastic/kibana/issues, https://github.com/elastic/beats/issuess, https://github.com/elastic/package-registry/issues, and contact us https://ela.st/ingest-manager-feedback[here]. Your feedback is very valuable to us.
+We have collected the most common known problems and frequently asked questions
+here. If your question isn't answered here, please review open issues in the
+following GitHub repositories:
+
+* https://github.com/elastic/kibana/issues[{kib}]
+* https://github.com/elastic/beats/issuess[{beats}]
+* https://github.com/elastic/package-registry/issues[{package-registry}]
+
+Contact us https://ela.st/ingest-manager-feedback[here]. Your feedback is very
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:19]:
```suggestion
Contact us in the https://ela.st/ingest-manager-feedback[discuss forum]. Your feedback is very
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 9]
```diff
@@ -4,72 +4,181 @@
...
+
+* https://github.com/elastic/kibana/issues[{kib}]
+* https://github.com/elastic/beats/issuess[{beats}]
+* https://github.com/elastic/package-registry/issues[{package-registry}]
+
+Contact us in the https://ela.st/ingest-manager-feedback[discuss forum]. Your feedback is very
+valuable to us.
+
+//REVIEWERS: Using "here" for link text is strongly discouraged because it's
+//bad for SEO. What text should we provide instead?
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:23]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 10]
```diff
@@ -4,72 +4,181 @@
...
+[[ingest-manager-not-in-kibana]]
+== The {ingest-manager} app is not listed in the {kib} side navigation
+
+In 7.8, the {ingest-manager} app is experimental. You must enable the app to
+see it in {kib}.
+
+//REVIEWERS: I don't want to have separate sections for cloud vs self-managed
+//because we are working on a widget for switching between instructions for
+//the two. Also I think it's more confusing to users if we have two sections
+//that cover the same content.
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:54]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 11]
```diff
@@ -4,72 +4,181 @@
...
+. Click **Save**.
+
+{kib} will restart automatically. When {kib} is available, refresh the browser
+to see the {ingest-manager} app in the navigation menu.
+
+**To enable {ingest-manager} on a self-managed cluster:**
+
+//REVIEWERS: Can't on-premise also mean cloud? I've seen self-managed used, too
+//so I've used that here. 
+
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:81]:
```suggestion
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 12]
```diff
@@ -78,81 +187,121 @@ elastic-agent.exe run
...
-[source,yaml]
-----
-xpack.ingestManager.enabled: true
-xpack.ingestManager.fleet.tlsCheckDisabled: true
-xpack.security.enabled: true
-elasticsearch.username: "elastic"
-elasticsearch.password: "abc123iUnbRftkABC123"
-----
+//REVIEWERS: Do we want to show the kill command. Not sure...if we do, which
+//kill command should we show?  kill -9? kill SIGTERM?
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:280]:
```suggestion
```
